File: students/students/src/MON.Models/Diploma/DiplomaTemplateListModel.cs
namespace MON.Models.Diploma
{
    public class DiplomaTemplateListModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string InstitutionName { get; set; }
        public string BasicDocumentTypeName { get; set; }
        public bool CanBeDeleted { get; set; }
        public string BasicClassName { get; set; }
    }
}

File: students/students/src/MON.SPA/src/models/diploma/diplomaTemplateListModel.js
export class DiplomaTemplateListModel {
    constructor(obj = {}) {
      this.id = obj.id || null;
      this.name = obj.name || null;
      this.institutionName = obj.institutionName || null;
      this.basicDocumentTypeName = obj.basicDocumentTypeName || null;
      this.canBeDeleted = obj.canBeDeleted || null;
    }
  }